### PR TITLE
Fixes new players being kicked for having an empty charactesheet

### LINKED ISSUE #8687: New Players are kicked for having no character sheet before they can make a valid one.
### Detailed description of bug

#8534 changed the order the character creation settings are done in order to better prepare for forks, but servers check for valid character sheets upon joining a server, which means if a player does not have one, they get kicked from the server before they have a chance to join and make one.

### PR DESCRIPTION
**Pull request by MaxIsJoe**
Fixes #8687

New players will now be automatically prompted to make a character when they join the server as well.

CL : [Fix] - Fixed new players being kicked due to the lack of a character sheet.
CL: [Improvement] - New players are automatically prompted to create a character sheet when first joining a server now.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Managers/PlayerList.Admin.cs [hunk 1]
```diff
@@ -433,10 +435,16 @@ private async Task<bool> CheckUserState(string unverifiedToken, ConnectedPlayer
...
-				Category.Admin);
+			var prefsCheck = PlayerPrefs.GetString("currentcharacter");
+			if (string.IsNullOrEmpty(prefsCheck))
+			{
+				StartCoroutine(KickPlayer(unverifiedConnPlayer, $"Server Error: Account has invalid username (Null/Empty). To fix go to character creator then click Serialise and then load"));
+				Logger.LogError($"A user tried to connect with null/empty username" +
+				                $"Details: Username: {unverifiedConnPlayer.Username}, ClientID: {unverifiedConnPlayer.ClientId}, IP: {unverifiedConnPlayer.ConnectionIP}",
+					Category.Admin);
+			}
+			LobbyManager.Instance.characterCustomization.SetActive(true);
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Managers/PlayerList.Admin.cs]: isn't this only server side?

File: UnityProject/Assets/Scripts/Managers/PlayerList.Admin.cs [hunk 2]
```diff
@@ -433,10 +435,16 @@ private async Task<bool> CheckUserState(string unverifiedToken, ConnectedPlayer
...
 		if (string.IsNullOrEmpty(unverifiedConnPlayer.Username))
 		{
-			StartCoroutine(KickPlayer(unverifiedConnPlayer, $"Server Error: Account has invalid username (Null/Empty). To fix go to character creator then click Serialise and then load"));
-			Logger.LogError($"A user tried to connect with null/empty username" +
-			           $"Details: Username: {unverifiedConnPlayer.Username}, ClientID: {unverifiedConnPlayer.ClientId}, IP: {unverifiedConnPlayer.ConnectionIP}",
-				Category.Admin);
+			var prefsCheck = PlayerPrefs.GetString("currentcharacter");
+			if (string.IsNullOrEmpty(prefsCheck))
+			{
+				StartCoroutine(KickPlayer(unverifiedConnPlayer, $"Server Error: Account has invalid username (Null/Empty). To fix go to character creator then click Serialise and then load"));
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Managers/PlayerList.Admin.cs]: and doesn't this still kick them?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Managers/PlayerList.Admin.cs]: only kicks if the prefs are empty.

File: UnityProject/Assets/Scripts/Managers/PlayerList.Admin.cs (hunk 1 above)
Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Managers/PlayerList.Admin.cs]: maybe? should I turn that into a net message?

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Managers/PlayerList.Admin.cs]: or rpc

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Managers/PlayerList.Admin.cs]: Done 👍

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**